Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+# Gopkg.toml example
```

Comment:
Do we need package manager here? I don't see any third-party packages you import.

Reply:
Thanks! 
I understood you)